frontend/server/src/DAO/Problems.php

```diff
@@ -1133,9 +1133,12 @@ public static function hasBeenUsedInCoursesOrContests(
         return $problems;
     }
 
-    final public static function getProblemsPerTagCount(string $problemLevelTag): int {
-        $sql = 'SELECT
-                    COUNT(Problems.problem_id) AS pa
+    /**
+     * @return list<\OmegaUp\DAO\VO\Problems>
+     */
+    final public static function getProblemsPerTagCount(): array {
+        $sql = "SELECT
+                    t.name, COUNT(p.problem_id) AS pa
```

Comment:
Por consistencia con la anotación que hiciste en L1158, puedes cambiar por
```suggestion
                    t.name,
                    COUNT(p.problem_id) AS problems_per_tag
```
?